src/main/java/Main.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int n, t; //자동차 개수랑 경기횟수
```

Comment:
```suggestion
        int numberOfCar; // 자동차 개수
        int trialCount; // 경기횟수
```
주석보단 이름을 통해 의미를 표현하는건 어떨까요?